live/config/environment.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (environment) {
     },
 
     googleFonts: [
-      'Lato', // main font
+      'Lato:400,700,900', // main font, Challenge instructions
```

Comment:
400 = default font (body, instructions...)
700 = some bold text (window comparison, warning mobile)
900 = only for strong emphasis (this screen and that screen)